### PR TITLE
Sign up design

### PR DESCRIPTION
**Pull request by tenzgass**
Add the different buttons (4)
Change variable colors

### CODE REVIEW

File: app/assets/stylesheets/components/_buttons.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.btn-orange {
+    background-color: $color-button-orange;
+    border-radius: 10px;
+    padding: 15px 15px;
```
Comment (Simon-Busch) [inline app/assets/stylesheets/components/_buttons.scss:4]: change to 1.5 rem

File: app/assets/stylesheets/components/_buttons.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.btn-orange {
+    background-color: $color-button-orange;
+    border-radius: 10px;
```
Comment (Simon-Busch) [inline app/assets/stylesheets/components/_buttons.scss:3]: change to 1rem

File: app/assets/stylesheets/components/_buttons.scss [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    font-size: 1.8rem;
+    color: white;
+    text-align: center;
+    display: inline-block;
+}
+
+.btn-yellow {
+    background-color: $color-button-yellow;
+    border-radius: 10px;
+    padding: 15px 15px;
```
Comment (Simon-Busch) [inline app/assets/stylesheets/components/_buttons.scss:17]: same

File: app/assets/stylesheets/components/_buttons.scss [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    font-size: 1.8rem;
+    color: white;
+    text-align: center;
+    display: inline-block;
+}
+
+.btn-next {
+    background-color: $color-button-orange;
+    border-radius: 10px;
+    padding: 15px 15px;
```
Comment (Simon-Busch) [inline app/assets/stylesheets/components/_buttons.scss:30]: same

File: app/assets/stylesheets/components/_buttons.scss [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    font-size: 1.8rem;
+    color: white;
+    text-align: center;
+    display: inline-block;
+}
+
+.btn-arrow {
+    background-color: $color-button-yellow;
+    border-radius: 10px;
+    padding: 15px 15px;
```
Comment (Simon-Busch) [inline app/assets/stylesheets/components/_buttons.scss:43]: same

**Review by Simon-Busch — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/components/_buttons.scss (hunk 5 above)
Comment (tenzgass) [reply app/assets/stylesheets/components/_buttons.scss:43]: done

File: app/assets/stylesheets/components/_buttons.scss (hunk 4 above)
Comment (tenzgass) [reply app/assets/stylesheets/components/_buttons.scss:30]: done

File: app/assets/stylesheets/components/_buttons.scss (hunk 3 above)
Comment (tenzgass) [reply app/assets/stylesheets/components/_buttons.scss:17]: done

File: app/assets/stylesheets/components/_buttons.scss (hunk 2 above)
Comment (tenzgass) [reply app/assets/stylesheets/components/_buttons.scss:3]: done

File: app/assets/stylesheets/components/_buttons.scss (hunk 1 above)
Comment (tenzgass) [reply app/assets/stylesheets/components/_buttons.scss:4]: done

**Pull request merged**